src/serializer.ts

```diff
@@ -56,7 +56,7 @@ class Serializer {
       return;
     }
 
-    var normalizedAccount = account;
+    let normalizedAccount = account;
```

Comment:
Why do you prefer `let` over `const` or `var` here?

Reply:
Best practice is to not use var anymore as let is a more strict option since it is block scoped instead of function scoped. Let is best for variables that change value like this one (line 71) where as const is best for variables that do not change value.

Reply:
When he says "`let` is best for variables that change value." he means, "You couldn't change the value of `normalizedAccount` like you do on line 71 if it was `const`."